app/src/main/res/values-fi/strings-fi.xml

```diff
@@ -204,10 +204,10 @@
 	<string name="wear_integration">Wear Integration</string>
 	<string name="send_data_to_android_wear_watch">Lähetä tietoja Android Wear Watchface.</string>
 	<string name="android_wear_integration">Android Wear Integration</string>
-	<string name="pref_wear_connectG5">Yhdistä Wear G5</string>
-	<string name="pref_summary_wear_connectG5">Yhdistä Wear suoraan G5</string>
-	<string name="pref_use_wear_connectG5">Käytä Wear G5 Collection Service</string>
-	<string name="pref_summary_use_wear_connectG5">Force puhelin käyttää Wear G5 Collection Service</string>
+	<string name="pref_enable_wearG5">Enable Wear Collection Service</string>
```

Comment:
Where a translation hasn't been made yes the strings should be missing from the localized file so that they default to the english strings. So I would delete these additions